web/app/routes/mainRoutes.js

```diff
@@ -0,0 +1,83 @@
+var express = require("express");
+var router = express.Router();
+var mongoose = require("mongoose");
+mongoose.connect("mongodb://localhost/routerdb");
```

Comment:
Leave this for now, but once you merge this in I'm going to move this to the server.js file and put the string into the config file I am building, considering we still don't have a production mongo server yet lol

Might also be worth adding that you need to have "Mongo server running locally required for development" to the README